### ISSUE TITLE
Remove extra white space between heading and description (seen in Twenty Twenty theme)

### ISSUE THREAD

**Issue by marklchaves** [marklchaves/will-work-for-ko-fi#3]
In the Twenty Twenty theme, there's too much white space (negative space) between the heading and description.

This is because the heading and description content are wrapped by `<section>` tags. In the Twenty Twenty theme, `<section>` tags have `5rem` top/bottom padding by default.

The temporary workaround is this:

```css
/** This temporarily fixes the issue. */
article.wp-block-cgb-block-will-work-for-ko-fi-cgb > section {
    	padding-top: 1rem;
    	padding-bottom: 1rem;
}
```

A better permanent solution would be something like this: 

```css
/* CSS */
.section—kofi {
    	padding-top: 1rem;
    	padding-bottom: 1rem;
}
```

```html
<!— HTML —>
<section class="section—kofi">
```

**Comment by marklchaves** [comment]
Added section__kofi class with default of 0 top/bottom padding.

https://github.com/marklchaves/will-work-for-ko-fi/commit/3e9aab3eaaac4abe1f77362f63bd8903401d41eb

**Comment by marklchaves** [comment]
Fixed in version 2.1.0.
